src/connectors/geo-search/connectGeoSearch.js

```diff
@@ -171,7 +171,10 @@ const connectGeoSearch = (renderFn, unmountFn) => {
 
     const toggleRefineOnMapMove = () =>
       widgetState.internalToggleRefineOnMapMove();
-    const createInternalToggleRefinementonMapMove = (render, args) => () => {
+    const createInternalToggleFacetRefinementOnMapMove = (
+      render,
+      args
+    ) => () => {
```

Comment:
This one should be reverted, because it's not related with facet refinement. The function change how the widget will trigger the refinement (on map move or not).